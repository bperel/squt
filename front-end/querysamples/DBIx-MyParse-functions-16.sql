SELECT COERCIBILITY('abc' COLLATE latin1_swedish_ci);

