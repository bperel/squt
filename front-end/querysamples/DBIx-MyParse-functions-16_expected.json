{
   "Warning" : {
      "No alias" : {
         "coercibility" : ["function", "SELECT"]
      }
   },
   "Functions" : {
      "1" : {
         "to" : "0",
         "Constants" : {
            "abc" : "abc",
            "latin1_swedish_ci" : "latin1_swedish_ci"
         },
         "name" : "collate",
         "alias" : "1"
      },
      "0" : {
         "to" : "OUTPUT",
         "name" : "coercibility",
         "alias" : "0"
      }
   }
}